pycc/cchbar.py

```diff
@@ -66,31 +65,59 @@ def __init__(self, ccwfn):
         L = ccwfn.H.L
         t1 = ccwfn.t1
         t2 = ccwfn.t2
-
-        self.Hov = self.build_Hov(o, v, F, L, t1)
-        self.Hvv = self.build_Hvv(o, v, F, L, t1, t2)
-        self.Hoo = self.build_Hoo(o, v, F, L, t1, t2)
-        self.Hoooo = self.build_Hoooo(o, v, ERI, t1, t2)
-        self.Hvvvv = self.build_Hvvvv(o, v, ERI, t1, t2)
-        self.Hvovv = self.build_Hvovv(o, v, ERI, t1)
-        self.Hooov = self.build_Hooov(o, v, ERI, t1)
-        self.Hovvo = self.build_Hovvo(o, v, ERI, L, t1, t2)
-        self.Hovov = self.build_Hovov(o, v, ERI, t1, t2)
-        self.Hvvvo = self.build_Hvvvo(o, v, ERI, L, self.Hov, self.Hvvvv, t1, t2)
-        self.Hovoo = self.build_Hovoo(o, v, ERI, L, self.Hov, self.Hoooo, t1, t2)
-    
-        if isinstance(t1, torch.Tensor):
-            print("Hov norm = %20.15f" % torch.linalg.norm(self.Hov))
-            print("Hvv norm = %20.15f" % torch.linalg.norm(self.Hvv))
-            print("Hoo norm = %20.15f" % torch.linalg.norm(self.Hoo))
-            print("Hoooo norm = %20.15f" % torch.linalg.norm(self.Hoooo))
-            print("Hvvvv norm = %20.15f" % torch.linalg.norm(self.Hvvvv))
-        else:
-            print("Hov norm = %20.15f" % np.linalg.norm(self.Hov))
-            print("Hvv norm = %20.15f" % np.linalg.norm(self.Hvv))
-            print("Hoo norm = %20.15f" % np.linalg.norm(self.Hoo))
-            print("Hoooo norm = %20.15f" % np.linalg.norm(self.Hoooo))
-            print("Hvvvv norm = %20.15f" % np.linalg.norm(self.Hvvvv))
+        
+        if ccwfn.local is None or ccwfn.filter is True:
```

Comment:
If I recall correctly, we discussed in our group meeting last week (5/16) that you were going to move the if-branches to the highest level and put all the `Local` calls for `ccwfn`, `cchbar`, and `cclambda` into a separate sub-directory.  Is that still the plan?  I don't remember if we decided you'd handle that **after** the current PR, so please remind me.

Reply:
Yes, this is the plan after the current PR.